static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+/* global google*/
+/*
+ Module: abtest-item.js
+ Description: Displays information about a single test
+ */
+import Component from 'common/modules/component';
+import Participation from 'admin/modules/abtests/participation';
+import bonzo from 'bonzo';
+import bean from 'bean';
+
+class ABTestReportItem extends Component {
+    constructor(config: any): void {
+        super();
+
+        this.config = Object.assign(Object.assign({}, this.config), config);
+
+        if (window.abCharts) {
+            this.chart = window.abCharts[`ab${this.config.test.id}`];
+        }
+    }
+
+    ready(): void {
+        if (this.chart) {
+            const redraw = this.renderChart.bind(this);
+            redraw();
+            let timerid;
+            bean.on(window, 'resize', () => {
```

Comment:
Could you add a comment here, that this should use lodashs `debounce` function instead?